src/vaadin-message.js

```diff
@@ -141,8 +156,23 @@ class MessageElement extends ElementMixin(ThemableMixin(PolymerElement)) {
       ></vaadin-avatar>
       <div part="content">
         <div part="header">
-          <span part="name">[[userName]]</span>
-          <span part="time">[[time]]</span>
+          <div part="sender">
+            <span part="name">[[userName]]</span>
+            <span part="time">[[time]]</span>
+          </div>
+          <vaadin-context-menu open-on="click">
+            <!-- When the menu opens, we should set aria-expanded to 'true' -->
+            <vaadin-button
+              aria-controls="vaadin-message-actions-menu"
+              aria-expanded="false"
+              aria-haspopup="true"
+              aria-label="Actions"
+              id="vaadin-message-actions-button"
+              theme="icon tertiary-inline"
+            >
+              <iron-icon aria-hidden="true" icon="vaadin:ellipsis-dots-v"></iron-icon>
+            </vaadin-button>
```

Comment:
Consider using the same DOM structure and styles as [vaadin-menu-bar overflow button](https://github.com/vaadin/vaadin-menu-bar/blob/fd7ebd5d34ca18684b32e72143f844c11fdb8071/src/vaadin-menu-bar.js#L88-L90).

Reply:
I'll have a look-see.